src/json-patch-queue.js

```diff
@@ -59,13 +59,13 @@ JSONPatchQueue.prototype.remoteVersion = 0;
  * @param  {Function} [applyCallback]     optional `function(object, consecutivePatch)` to be called when applied, if not given #apply will be called
  */
 JSONPatchQueue.prototype.receive = function(versionedJsonPatch, applyCallback){
-	var apply = applyCallback || this.apply,
-		consecutivePatch = versionedJsonPatch.slice(0);
+	const apply = applyCallback || this.apply;
+	let consecutivePatch = versionedJsonPatch.slice(0);
 	// strip Versioned JSON Patch specyfiv operation objects from given sequence
 		if(this.purist){
-			var testRemote = consecutivePatch.shift();
+			const testRemote = consecutivePatch.shift();
```

Comment:
This has to be defined outside of the `if`s block, otherwise it is not available to the script outside of the `if`.